chain/client/src/sync.rs

```diff
@@ -52,19 +52,12 @@ pub const MAX_PENDING_PART: u64 = MAX_STATE_PART_REQUEST * 10000;
 pub const NS_PER_SECOND: u128 = 1_000_000_000;
 
 /// Get random peer from the hightest height peers.
-pub fn highest_height_peer(
-    highest_height_peers: &Vec<FullPeerInfo>,
-    min_height: BlockHeight,
```

Comment:
why is this removed?

Reply:
It was a workaround to make sure the syncing doesn't end prematurely (in the old approach once we start syncing, the code could choose a peer that is, same as us, one height behind and immediately stop syncing).